api/api.proto

```diff
@@ -932,6 +932,50 @@ service OpenStorageCluster {
         get: "/v1/clusters/current"
       };
     }
+    // Creates Pair with a remote cluster and returns details about the remote cluster
```

Comment:
Please create a new service called `OpenStorageClusterPair` to group these functions, that way it will eliminate the name collision problem.

Reply:
Please add adequate documentation for non-openstorage users to understand.